Workbooks/AKS/Syslog/Syslog.workbook

```diff
@@ -0,0 +1,568 @@
+                "value::1"
+              ],
+              "showDefault": false
+            },
+            "defaultValue": "value::1"
+          },
+          {
+            "id": "5fea96dc-607c-450d-bc2e-ecb0756dc169",
+            "version": "KqlParameterItem/1.0",
+            "name": "ResourceType",
+            "type": 7,
+            "isRequired": true,
+            "query": "{\"version\":\"1.0.0\",\"content\":\"\\\"{Resource:resourceType}\\\"\",\"transformers\":null}",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::1"
+              ],
+              "showDefault": false
+            },
+            "defaultValue": "value::1",
+            "queryType": 8
+          },
+          {
+            "id": "9b289ad1-7eaa-411d-b2b0-43c69cf5aa14",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "isRequired": true,
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 300000
+                },
+                {
+                  "durationMs": 900000
+                },
+                {
+                  "durationMs": 1800000
+                },
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 43200000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 172800000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 1209600000
+                },
+                {
+                  "durationMs": 2419200000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                },
+                {
+                  "durationMs": 7776000000
+                }
+              ]
+            },
+            "value": {
+              "durationMs": 604800000
+            }
+          },
+          {
+            "id": "0698db8c-7a3a-4aec-bfb2-eb59942b0375",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "resources\r\n| where type =~ '{ResourceType}'\r\n| where name == \"{Resource:Name}\"\r\n| extend workspaceResourceId = tostring(parse_json(tolower(properties)).addonprofiles.omsagent.config.loganalyticsworkspaceresourceid)\r\n| project workspaceResourceId ",

```

Comment:
where name **==** \"{Resource:Name} should be where name =~ for case insensitive comparison

Reply:
Update